docusaurus/docs/getting-started.mdx

```diff
@@ -42,12 +42,6 @@ Make sure you are using supported a supported OS, Grafana version, and tooling.
 
 #### Supported operating systems
 
-- [Go](https://go.dev/doc/install) version 1.18 or above
```

Comment:
Thanks so much for spotting this documentation issue @slorello89. ⭐ 

From looking at the two "similar" lists of content I think rather than delete this list it should be moved to replace the list under `Recommended tooling` [here](https://github.com/grafana/plugin-tools/pull/284/files#diff-d15c44113d543f0a9d6bb9cc5dbbdbdebb7341284d1f45109104fed67a878b5aL65-L70). Would it be possible for you to update this PR in this way?

Reply:
no problem.